R/bootstrap.R

```diff
@@ -44,6 +45,19 @@ bootstrapPage <- function(..., title = NULL, responsive = NULL, theme = NULL) {
     ),
     bootstrapLib()
   )
+
+  # Also store `lang` to be passed for rendering processor.
+  if (!is.null(lang)) {
+    if (is.character(lang) && length(lang) == 1) {
+      # Append lang attribute to be passed to renderPage function
+      attributes(ui) <- c(attributes(ui), list(lang = lang))
+    } else {
+      warning("Invalid lang value: defaulting to empty string.")
+      attributes(ui) <- c(attributes(ui), list(lang = NULL))
```

Comment:
Can we remove this line? Then the behavior would be consistent with the case where `lang` actually is `NULL`.